config/runner/toolkit.yml

```diff
@@ -15,6 +15,12 @@ toolkit:
     asda_source: 'reference'
     myloader: '/scripts/importdb.sh'
     mydumper: '/scripts/backupdb.sh'
+  forbidden_files:
+    - name: ".env"
+      error: ".env file detected!\n In order to avoid loading .env file in ACC or PROD environments, please rename it to something like .env.dist\n The file belongs to your application. You should guarantee that is not being called into any point of your Drupal site. If you are only calling it in docker-compose.yml, it's recommended to drop entirely the file and declare the environment variables straight into docker-composer.yml file."
+    - name: "grumphp.yml.dist"
+      error: "grumphp.yml.dist file detected!\n The usage of GrumPHP was removed in the upgrade for ec-europa/toolkit:^9, please remove grumphp.yml.dist file."
```

Comment:
I would say this in a different way.
Something like: 

The grumphp.yml.dist file was detected but the package is not being required by your project, please remove the file.

I think is not important to know when it was removed, but instead, would be good to tell them that they are not using a package but they have configuration for it.

Reply:
Hi @joaocsilva it's fixed.